src/activities/pdfUtils.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Converts a color hex code into RGBA values.
+ * @param hex The color hex code.
+ * @returns RGBA values between 0 and 1.
+ */
+export function hexToRgba(hex = "000000FF") {
+    hex = hex.replace("#", "");
+    const red = parseInt(hex.substring(0, 2), 16) / 255;
+    const green = parseInt(hex.substring(2, 4), 16) / 255;
+    const blue = parseInt(hex.substring(4, 6), 16) / 255;
+    const alpha = hex.length > 6 ? parseInt(hex.substring(6, 8), 16) / 255 : 1;
```

Comment:
I think that we should match the changes made in the typescript implementation. We should only allow 6 or 8 characters.

Reply:
Done